### PR TITLE
Task-49792 : Remove log4j dependency from test and compile scopes to exclusively use the same logback logging framework as in runtime

### PR DESCRIPTION
**Pull request by rdenarie**

### CODE REVIEW

File: test/core/pom.xml [hunk 1]
```diff
@@ -14,37 +14,36 @@
...
+           <groupId>org.exoplatform.gatein.pc</groupId>
+           <artifactId>pc-portlet</artifactId>
+       </dependency>
+       <dependency>
+           <groupId>org.exoplatform.gatein.pc</groupId>
+           <artifactId>pc-api</artifactId>
+       </dependency>
+       <dependency>
+           <groupId>org.exoplatform.kernel</groupId>
+           <artifactId>exo.kernel.commons</artifactId>
```
Comment (boubaker) [inline test/core/pom.xml]: added twice?

File: test/core/pom.xml [hunk 2]
```diff
@@ -14,37 +14,36 @@
...
-         <groupId>commons-logging</groupId>
-         <artifactId>commons-logging</artifactId>
-      </dependency>
-      <dependency>
-         <groupId>commons-httpclient</groupId>
-         <artifactId>commons-httpclient</artifactId>
-      </dependency>
+       <!-- Various dependencies -->
+       <dependency>
+           <groupId>commons-logging</groupId>
```
Comment (boubaker) [inline test/core/pom.xml]: those two common libraries should come transitively from `exo.kernel.commons`, isn't it ?

File: test/core/pom.xml (hunk 1 above)
Comment (rdenarie) [reply test/core/pom.xml]: removed

File: test/core/pom.xml (hunk 2 above)
Comment (rdenarie) [reply test/core/pom.xml]: yes, removed

**Review by boubaker — APPROVED** [review]

**Pull request merged**